pkg/ovs/openflow/utils.go

```diff
@@ -949,6 +966,13 @@ func getFlowModMatch(flowMod *openflow15.FlowMod) string {
 		}
 	}
 
+	// xxregs
+	for i := 0; i < 4; i++ {
+		if field, ok := matchMap[fmt.Sprintf("xxreg%d", i)]; ok {
+			parts = append(parts, matchXXRegToString(i, field))
+		}
+	}
+
 	// TODO: other match conditions about tun
 	// tun_dst/tun_ipv6_dst
```

Comment:
This can be removed?

Reply:
I think we can keep comments these to identify which OVS fiels the logic is targeting at?

Reply:
Not suggested, easy to be out-of-date. You added a new `tun_id` condition but forget to update this line.
Not sure if this `TODO: other match conditions about tun` comment helps.

Reply:
removed "tun_dst/tun_ipv6_dst". I left the "TODO...." here because this file is used to generate flow string following the order of OVS ofctl commands. By now we only add logic for the fields used in antrea. If we introduce more fields in the future, we should ensure the new fields are translated in the correct value, so the "TODO" here are for referrence.